setup.py

```diff
@@ -11,6 +11,7 @@
         'AuvHandler = aodndata.auv.handler:AuvHandler',
         'ArgoHandler = aodndata.argo.handler:ArgoHandler',
         'AsyncUploadHandler = aodndata.common.asyncupload:AsyncUploadHandler',
+        'ImosBgcDbHandler = aodndata.imos_bgc_db.handler:ImosBgcDbHandler',
```

Comment:
is "Imos..." a valid handler name considering that non of the other IMOS handlers have this string in?

Reply:
It's called that because it's a handler for the "IMOS BGC Database" project (aka Octopus, but I resisted the urge to call it that). That's what CSIRO called the project, so I've just stuck with that (the db schema is `imos_bgc_db`)

Reply:
ok, but then needs to be ordered alphabetically as with other handlers in the list

Reply:
Ah, good point. Done.